src/relay/WalletSDKRelay.ts

```diff
@@ -707,8 +701,8 @@ export class WalletSDKRelay extends WalletSDKRelayAbstract {
       origin: location.origin
     })
     if (isRequestEthereumAccountsResponse(response)) {
-      Array.from(WalletSDKRelay.accountRequestCallbackIds.values()).forEach(
-        id => this.invokeCallback({ ...message, id })
+      WalletSDKRelay.accountRequestCallbackIds.forEach(
+        (_, id) => this.invokeCallback({ ...message, id })
```

Comment:
😄

Reply:
Reciver
ID
Ledengineeringsupply@gmail.com​

Reply:
since it's a set, you can do:
```suggestion
        (id) => this.invokeCallback({ ...message, id })
```